cgrok/grokre.c

```diff
@@ -404,20 +404,38 @@ static void grok_study_capture_map(grok_t *grok) {
 
   /* Grok creates PCRE named groups where the name is a hex number.
      Then it tries to extract the number from the PCRE nametable, 
-     so as to tie the index of the PCRE group to the Grok variable name.
-
-     PCRE named groups like (?<group>.*) will be ignored.
-      */ 
+     so as to tie the index of the PCRE group to the Grok variable name.*/ 
   for (i = 0; i < nametable_size; i++) {
     offset = i * nametable_entrysize;
     stringnum = (nametable[offset] << 8) + nametable[offset + 1];
+    char *groupName = nametable + offset + 2;
+    capture_id = -1;
     sscanf(nametable + offset + 2, CAPTURE_FORMAT, &capture_id);
```

Comment:
Since we define `groupName` as `nametable + offset + 2`, could we use it in the `sscanf` call too?